### PR TITLE
Log policy update event errors and silently delete bwmap entries

### PR DESCRIPTION
**Pull request by zuzzas**
Everything is properly logged or passed up the call stack if there is an error in the result channel that is sent over an EventQueue.

Except these 3 methods in this PR.

While adding error logging, I've found the source of false entries. We are **always** [deleting](https://github.com/cilium/cilium/blob/e75d6d0de4c466b8c6f97130e386d7019b4f406b/pkg/endpoint/events.go#L291) a bwmap entry. And if we are doing that... why are we interested in ENOENT?

```release-notes
Log policy update event errors and silently delete bwmap entries on event handling
```

### CODE REVIEW

**Review by aanm — APPROVED** [review]

**Comment by zuzzas** [comment]
@aanm 
I've resolved an issue with CI tests that surfaced after surfacing the BandwidthError in the log. Please, retest, and let me know if you are okay with the solution (in a separate commit).

**Review by aanm — APPROVED** [review]

**Review by YutaroHayakawa — APPROVED** [review]

File: pkg/maps/bwmap/bwmap.go [hunk 1]
```diff
@@ -66,3 +66,14 @@ func Delete(Id uint16) error {
...
+
+func SilentDelete(Id uint16) error {
+	_, err := ThrottleMap.SilentDelete(
+		&EdtId{Id: uint64(Id)})
+
+	if err != nil {
+		return err
+	}
+
+	return nil
```
Comment (jrajahalme) [inline pkg/maps/bwmap/bwmap.go]: nit: just `return err` here?

**Review by jrajahalme — APPROVED** [review]
Thanks for the fix, a minor nit only you may want to address before we merge.

File: pkg/maps/bwmap/bwmap.go (hunk 1 above)
Comment (zuzzas) [reply pkg/maps/bwmap/bwmap.go]: Obviously! Fixed and re-pushed.
https://github.com/cilium/cilium/compare/909261162597ce88bb53e44f253c6f8fd4418097..280de074362d21880b247c95674c33215175671a

**Comment by aanm** [comment]
The CI was green before and the only change was the one pointed out [here](https://github.com/cilium/cilium/compare/909261162597ce88bb53e44f253c6f8fd4418097..280de074362d21880b247c95674c33215175671a) Merging the PR

**Pull request merged**